### ISSUE TITLE
CT phetioID already registered

### ISSUE THREAD

**Issue by KatieWoe** [phetsims/molecules-and-light#375]
```
molecules-and-light : phet-io-state-fuzz : unbuilt
https://bayes.colorado.edu/continuous-testing/ct-snapshots/1601845816475/phet-io-wrappers/state/?sim=molecules-and-light&phetioDebug&fuzz&wrapperContinuousTest=%7B%22test%22%3A%5B%22molecules-and-light%22%2C%22phet-io-state-fuzz%22%2C%22unbuilt%22%5D%2C%22snapshotName%22%3A%22snapshot-1601845816475%22%2C%22timestamp%22%3A1601880793620%7D
Uncaught Error: Uncaught Error: Assertion failed: phetioID already registered: moleculesAndLight.moleculesAndLightScreen.model.photons~16.positionProperty
Error: Assertion failed: phetioID already registered: moleculesAndLight.moleculesAndLightScreen.model.photons~16.positionProperty
at window.assertions.assertFunction (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1601845816475/assert/js/assert.js:22:13)
at PhetioEngine.phetioObjectAdded (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1601845816475/phet-io/js/phetioEngine.js:432:36)
at Object.addPhetioObject (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1601845816475/phet-io/js/phetioEngine.js:1047:49)
at Tandem.addPhetioObject (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1601845816475/tandem/js/Tandem.js:171:38)
at Vector2Property.initializePhetioObject (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1601845816475/tandem/js/PhetioObject.js:343:17)
at new PhetioObject (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1601845816475/tandem/js/PhetioObject.js:189:10)
at new Property (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1601845816475/axon/js/Property.js:72:5)
at new Vector2Property (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1601845816475/dot/js/Vector2Property.js:43:5)
at new Photon (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1601845816475/molecules-and-light/js/photon-absorption/model/Photon.js:30:27)
at MoleculesAndLightModel.emitPhoton (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1601845816475/molecules-and-light/js/photon-absorption/model/PhotonAbsorptionModel.js:342:20)
id: Bayes Chrome
Snapshot from 10/4/2020, 3:10:16 PM

----------------------------------

molecules-and-light : phet-io-state-fuzz : unbuilt
https://bayes.colorado.edu/continuous-testing/ct-snapshots/1601845816475/phet-io-wrappers/state/?sim=molecules-and-light&phetioDebug&fuzz&wrapperContinuousTest=%7B%22test%22%3A%5B%22molecules-and-light%22%2C%22phet-io-state-fuzz%22%2C%22unbuilt%22%5D%2C%22snapshotName%22%3A%22snapshot-1601845816475%22%2C%22timestamp%22%3A1601887830852%7D
Uncaught Error: Uncaught Error: Assertion failed: phetioID already registered: moleculesAndLight.moleculesAndLightScreen.model.photons~11.positionProperty
Error: Assertion failed: phetioID already registered: moleculesAndLight.moleculesAndLightScreen.model.photons~11.positionProperty
at window.assertions.assertFunction (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1601845816475/assert/js/assert.js:22:13)
at PhetioEngine.phetioObjectAdded (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1601845816475/phet-io/js/phetioEngine.js:432:36)
at Object.addPhetioObject (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1601845816475/phet-io/js/phetioEngine.js:1047:49)
at Tandem.addPhetioObject (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1601845816475/tandem/js/Tandem.js:171:38)
at Vector2Property.initializePhetioObject (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1601845816475/tandem/js/PhetioObject.js:343:17)
at new PhetioObject (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1601845816475/tandem/js/PhetioObject.js:189:10)
at new Property (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1601845816475/axon/js/Property.js:72:5)
at new Vector2Property (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1601845816475/dot/js/Vector2Property.js:43:5)
at new Photon (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1601845816475/molecules-and-light/js/photon-absorption/model/Photon.js:30:27)
at MoleculesAndLightModel.emitPhoton (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1601845816475/molecules-and-light/js/photon-absorption/model/PhotonAbsorptionModel.js:342:20)
id: Bayes Chrome
Snapshot from 10/4/2020, 3:10:16 PM
```

**Comment by samreid** [comment]
This will likely be solved by https://github.com/phetsims/molecules-and-light/issues/353

**Comment by samreid** [comment]
Proposed fix committed for #353, let's track it there.  Closing.
